<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DemGeneratorDialog</class>
 <widget class="QDialog" name="DemGeneratorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ìˆ˜ì¹˜ì§€í˜•ë„ DEM ìƒì„±ê¸° (DEM Generator) - ArchToolkit</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_Header">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;ê³ í•´ìƒë„ ì§€í˜• ëª¨ë¸ë§ ë„êµ¬&lt;/span&gt;&lt;/p&gt;&lt;p&gt;ìˆ˜ì¹˜ì§€í˜•ë„(DXF)ì˜ ë“±ê³ ì„  ë° í‘œê³ ì ì„ ë³´ê°„í•˜ì—¬ ì •ë°€í•œ DEMì„ ìƒì„±í•©ë‹ˆë‹¤. &lt;br/&gt;TIN(ë¶ˆê·œì¹™ ì‚¼ê°ë§) ë°©ì‹ì€ ë“±ê³ ì„  ë°ì´í„°ì— ê°€ì¥ ì í•©í•œ ë³´ê°„ ì•Œê³ ë¦¬ì¦˜ì…ë‹ˆë‹¤.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string>background-color: #e3f2fd; padding: 10px; border-radius: 4px; border: 1px solid #bbdefb;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupDxfLoad">
     <property name="title">
      <string>1. ìˆ˜ì¹˜ì§€í˜•ë„(DXF) ë¡œë“œ ë° ë ˆì´ì–´ í•„í„°ë§</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableWidget" name="tblLayers">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btnSelectAll">
          <property name="text">
           <string>ëª¨ë‘ ì„ íƒ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDeselectAll">
          <property name="text">
           <string>ì„ íƒ í•´ì œ</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnLoadDxf">
          <property name="text">
           <string>ğŸ“ DXF íŒŒì¼(ì—¬ëŸ¬ ê°œ ê°€ëŠ¥) ë¶ˆëŸ¬ì˜¤ê¸°...</string>
          </property>
          <property name="styleSheet">
           <string>font-weight: bold; padding: 5px 15px;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupVectorSelect">
     <property name="title">
      <string>2. ë³´ê°„ ëŒ€ìƒ ë ˆì´ì–´ í™•ì¸</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_Info2">
        <property name="text">
         <string>QGISì— ë¶ˆëŸ¬ì™€ì§„ ë ˆì´ì–´ ì¤‘, DEM ìƒì„±ì— ì‚¬ìš©í•  ë²¡í„° ë ˆì´ì–´ë¥¼ ì²´í¬í•˜ì„¸ìš”.</string>
        </property>
        <property name="styleSheet">
         <string>color: #666; font-style: italic;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listLayers">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>120</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRefreshLayers">
        <property name="text">
         <string>ğŸ”„ ë ˆì´ì–´ ëª©ë¡ ìƒˆë¡œê³ ì¹¨</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupConfig">
     <property name="title">
      <string>3. ë³´ê°„ ì•Œê³ ë¦¬ì¦˜ ë° í•´ìƒë„ ì„¤ì •</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblScale">
        <property name="text">
         <string>ë„ë©´ ì¶•ì²™ ê¸°ì¤€:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cmbScale">
        <property name="toolTip">
         <string>êµ­í† ì§€ë¦¬ì •ë³´ì› ë‚´ê·œì— ë”°ë¥¸ ì¶•ì²™ë³„ ê¶Œì¥ í•´ìƒë„ë¥¼ ìë™ ì…ë ¥í•©ë‹ˆë‹¤.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblPixelSize">
        <property name="text">
         <string>ì¶œë ¥ í•´ìƒë„ (m):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QDoubleSpinBox" name="spinPixelSize">
          <property name="value">
           <double>1.0</double>
          </property>
          <property name="singleStep">
           <double>0.5</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblRecommended">
          <property name="text">
           <string>(ê¶Œì¥: 1.0m)</string>
          </property>
          <property name="styleSheet">
           <string>color: #007bff; font-weight: bold;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblInterpolation">
        <property name="text">
         <string>ë³´ê°„ ë°©ë²• (Method):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cmbInterpolation"/>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="lblInterpDesc">
        <property name="text">
         <string>ğŸ’¡ TIN (Triangulated Irregular Network): ë¶ˆê·œì¹™ ì‚¼ê°ë§ ë³´ê°„ë²• [Delaunay, 1934]</string>
        </property>
        <property name="styleSheet">
         <string>background-color: #fff3e0; padding: 8px; border-radius: 3px; font-style: italic;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupOutput">
     <property name="title">
      <string>4. ê²°ê³¼ íŒŒì¼ ì €ì¥</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QgsFileWidget" name="fileOutput">
        <property name="dialogTitle">
         <string>DEM ì €ì¥ ìœ„ì¹˜ ì„ íƒ (GeoTIFF)</string>
        </property>
        <property name="filter">
         <string>GeoTIFF Files (*.tif)</string>
        </property>
        <property name="storageMode">
         <enum>QgsFileWidget::SaveFile</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnRun">
     <property name="text">
      <string>ğŸš€ DEM ìƒì„± ì‹œì‘</string>
     </property>
     <property name="styleSheet">
      <string>padding: 12px; font-size: 11pt; font-weight: bold; background-color: #4caf50; color: white; border-radius: 5px;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnClose">
     <property name="text">
      <string>ë‹«ê¸°</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgis.gui</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
